qgis4/ansible/Vagrantfile

```diff
@@ -0,0 +1,20 @@
+Vagrant.configure(2) do |config|
+
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 4
+  end
+
+  config.vm.box = "debian/stretch64"
+
+  config.vm.define "qgis4" do |machine|
+    machine.vm.hostname = "qgis4"
+    machine.vm.network "private_network", ip: "192.168.77.2"
+  end
+
+  config.vm.provision "ansible" do |ansible|
+    ansible.become = true
```

Comment:
`become` is already set to `true` in the playbook.

Reply:
Honestly I copy/pasted the vagrantfile from another project, so it is possibly not necessary

Reply:
I personally hate unnecessary (a.k.a. confusing) stuff :-)